### ISSUE TITLE
[Task]: Add Shimmer EVM Support

### ISSUE THREAD

**Issue by begonaalvarezd** [iotaledger/evm-toolkit#19]
We should add to the project ShimmerEVM support. 
Until the EVM launches, we should work with a a testnet chain, provided in slack (should not be pushed)
- [ ] Display the faucet tab or not based on if the selectedNetwork has faucet or not configured
- [ ] Update encoding (new encoding only for shimmer evm, not for shimmer evm testnet)
- [ ] Add final ShimmerEVM details when ready (blocked by release date)

#### Details about the new encoding:

ShimmerEVM (production) encoding change: 
basically we changed the way we identify "sender contracts" on  L1 requests
how does firefly encode this right now?
I'm guessing at this time is a uint32 (4 bytes) as 0s, right?
now just 1 byte (as 0) is needed, that's it
- a single 0 byte (means  no contract)
- a single byte with value 1, followed by 4 bytes (hname) - (this is the same as it was before, just with the "kind" byte added as a prefix)
- a single byte with value 2, followed by 20 bytes (EVM address)

**Comment by begonaalvarezd** [comment]
Closed by https://github.com/iotaledger/evm-toolkit/pull/21
